karspexet/ticket/templates/seat_map.svg

```diff
@@ -1,6 +1,7 @@
-<svg viewBox="0 0 400 480" xmlns="http://www.wd.org/2000/svg">
-  <rect x="0" y="400" width="390" height="40" class="stage"/>
-  <text x="200" y="420" width="390" class="stage-text">Scen</text>
+<svg viewBox="0 0 520 620" xmlns="http://www.wd.org/2000/svg">
```

Comment:
These numbers seem very magical, are they related to Södran?

Reply:
Sort of, yeah. They happen to match the x/y positions on the seats on Södran in the old ticket system. I wrote a small script to import those, then the proportions looked completely out of whack. Maybe we should have some `seat_map_dimensions` fields on the `Venue` model so we don't have to have these magical numbers here? On the other hand, I think I use these magic numbers in the css as well, and that's a bit hander to change. It can probably be solved in some other way though.

Reply:
I really want to extract those, but I think I'll maybe do that in a separate PR?

The old numbers are in a fashion just as magical, or at least arbitrary, since I made them up when I first implemented the seat selection.

Reply:
Yeah, a separate PR is fine. I suggest using those stored dimensions to render a style tag. The CSS wasn't very big anyways.